src/transformers/models/llava_next/modeling_llava_next.py

```diff
@@ -509,6 +509,19 @@ def _merge_input_ids_with_image_features(
         image_token_index = image_token_index if image_token_index is not None else self.config.image_token_index
         ignore_index = ignore_index if ignore_index is not None else self.config.ignore_index
 
+        if self.training and self.padding_side == "left":
+            logger.warning_once(
+                "Padding side is set to 'left' but the model is in training mode. For training "
+                "it is recommended to set `model.padding_side='right' and `processor.tokenizer.padding_side='right'`. "
+                "If that's intended, ignore this warning"
```

Comment:
Let's see how we get on here and if anyone complains - emitting warnings when there's no action to take can be annoying for users

Reply:
but isn't this whole field just - seeing warnings that you can't do anything about in your logs during init??

Reply:
I'm not really sure what you mean by "whole field". 

In general warnings should be used sparingly and there should be something the user can do to remove them -- this applies most of the time here -- so I think it's OK. I don't expect this to cause much issue - but experience has shown that people get annoyed if there's sudden, new warning messages they can't remove. Using `warning_once` should mitigate some of this